packages/repo-tools/package.json

```diff
@@ -74,6 +74,7 @@
     "just-diff": "^6.0.2",
     "knex": "^3.0.0",
     "knex-pglite": "^0.11.0",
+    "knip": "^5.42.0",
```

Comment:
The thing about having it in the root instead of here, is that we don't impose the cost of installing knip if you don't use it. The feature isn't really "external" fully yet; it's a bit hidden away and it's mostly us that use it inside the main repo, except some brave souls.

I don't have a very strong opinion here. What do you think?

Reply:
I don't have a strong opinion, I tested it the other way around and it doesn't seem to cause any harm. Happy to go either way.

Reply:
Yepyep it's not overly large, we can keep this.